tests/main/snap-connect/task.yaml

```diff
@@ -1,5 +1,11 @@
 summary: Check that snap connect works
 
+details: |
+    This test verifies multiple way to invoke "snap connect". It
+    checks proper errors are reported. It makes sure slot in some
+    contexts can be implicit. Calls are indempotent. It tests
```

Comment:
indempotent?

Reply:
Connecting an already connected connection does not do anything.

Reply:
idempotent. Sorry, that was a typo.